config/secrets.yml

```diff
@@ -17,16 +17,18 @@
 
 # Environmental secrets are only available for that specific environment.
 
-development:
-  secret_key_base: ffa096348fa590e131d67e30b0c3815610b7679a8bbf87e1bebff212e649d965585fccf955165d5999dfb562018d97892c5d217686fec21fee9e07943a8c4c11
 
-test:
-  secret_key_base: b13aa68d17a60b96bd5ed46033e75f9d2c327c7cc53335d3faef95d7409be140c779efbe367cc4f88cd4221c1d90009be39a5bf6f77da6e0ec33dc06917d489e
 
 # Do not keep production secrets in the unencrypted secrets file.
 # Instead, either read values from the environment.
 # Or, use `bin/rails secrets:setup` to configure encrypted secrets
 # and move the `production:` environment over there.
 
 production:
-  secret_key_base: <%= ENV["SECRET_KEY_BASE"] %>
+  secret_key_base: <%= Figaro.env.secret_key_base %>
```

Comment:
@s-espinosa @icorson3 
I think this is correct, but I'm not sure how to input the AWS variables that heroku needs. Or they might not need to be saved here because I'm also setting them using terminal commands heroku config AWS_THINGY=random5stuff3